x11-packages/rnote/build.sh

```diff
@@ -12,11 +13,7 @@ TERMUX_PKG_PYTHON_BUILD_DEPS="toml2json"
 
 __fetch_gettext_rs() {
 	# Latest version of gettext-sys, provided by the gettext-rs crate
-	local crate_version
-	crate_version="$(curl -s https://crates.io/api/v1/crates/gettext-sys | jq --raw-output '.crate.max_stable_version')"
```

Comment:
This was meant to help with auto updates.
What's the issue this is causing?

Reply:
You need to keep update the checksum as time goes on
Currently its evaluate to 0.22.5 and broke the build because mismatch checksum that was for 0.21.4